### ISSUE TITLE
[Facebook] hybridauth only displayname have value

### ISSUE THREAD

**Issue by ezekel** [hybridauth/hybridauth#528]
Hi I have problem the return data is this but the value of other fields are empty or null.what is the problem with this ?

```
object(Hybrid_User_Profile)#10 (24) { ["identifier"]=> string(17) "10207719888888" ["webSiteURL"]=> string(0) "" ["profileURL"]=> string(0) "" ["photoURL"]=> string(73) "https://graph.facebook.com/155550771908888/picture?width=150&height=150" ["displayName"]=> string(14) "My Name" ["description"]=> string(0) "" ["firstName"]=> string(0) "" ["lastName"]=> string(0) "" ["gender"]=> string(0) "" ["language"]=> string(0) "" ["age"]=> NULL ["birthDay"]=> NULL ["birthMonth"]=> NULL ["birthYear"]=> NULL ["email"]=> string(0) "" ["emailVerified"]=> string(0) "" ["phone"]=> NULL ["address"]=> NULL ["country"]=> NULL ["region"]=> string(0) "" ["city"]=> NULL ["zip"]=> NULL ["username"]=> string(0) "" ["coverInfoURL"]=> string(255) "https://graph.facebook.com333333333339968?fields=cover&access_token=xxxxxxxxxxxx" } 



```

Thank you in advance.

**Comment by AdwinTrave** [comment]
Check the newest code after #525 and see if that fixes the issue.

**Comment by AdwinTrave** [comment]
Are you using HybridAuth via a plugin?

**Comment by TheJoin95** [comment]
For v 2.4 Facebook api version  you must update your hybridauth library to last version (2.5)

**Comment by noneym** [comment]
i hava same problem, i want to take facebook username but not working on hybridauth 2.5.0

**Comment by ApacheEx** [comment]
pls try the latest version of HA (v2.9.5), there were so many fixes in FB. 
if something will not work, create a new issue or reopen existing one.
